src/common/low_precision_transformations/include/low_precision/markup_precisions.hpp

```diff
@@ -38,13 +38,26 @@ class ngraph::pass::low_precision::MarkupPrecisions : public ngraph::pass::Funct
 public:
     class Restriction {
     public:
+        class RestrictionByVersion {
+        public:
+            RestrictionByVersion() = default;
+            RestrictionByVersion(
+                const std::function<PrecisionsRestriction::PrecisionsByPorts(const std::shared_ptr<Node>&)>& precisionsFunction,
+                const PrecisionsRestriction::PrecisionsByPorts& precisions) :
+                precisionsFunction(precisionsFunction),
+                precisions(precisions) {}
+
```

Comment:
Can we introduce a method that will return `PrecisionsRestriction::PrecisionsByPorts` for a node depending on class fields?

Reply:
done: `get`